server/src/socket.js

```diff
@@ -32,30 +32,41 @@ export default class Socket {
     const json = {
       ...room,
       updatedAt: Date.now(),
-    }
+    };
 
-    return getRedis().hsetAsync('rooms', this._roomId, JSON.stringify(json))
+    return getStore().set('rooms', this._roomId, JSON.stringify(json));
   }
 
   async destroyRoom() {
-    return getRedis().hdel('rooms', this._roomId)
+    return getStore().del('rooms', this._roomId);
   }
 
   fetchRoom() {
     return new Promise(async (resolve, reject) => {
-      const res = await getRedis().hgetAsync('rooms', this._roomId)
-      resolve(JSON.parse(res || '{}'))
-    })
+      const res = await getStore().get('rooms', this._roomId);
+      resolve(JSON.parse(res || '{}'));
+    });
   }
 
-  joinRoom(roomId, socketId) {
+  joinRoom(roomId, socket) {
     return new Promise((resolve, reject) => {
-      getIO().of('/').adapter.remoteJoin(socketId, roomId, (err) => {
-        if (err) {
-          reject()
-        }
-        resolve()
-      });
+      if (process.env.REDIS_URL !== undefined) {
```

Comment:
Let's extend our store class to see if an adapter was initialized so we're not assuming that if a `STORE_HOST` is set, it is available. Then we are able to do `getStore().hasAdapter` and make decisions from there.